src/Metadata/Seabug.purs

```diff
@@ -205,5 +202,5 @@ mkKey :: Partial => String -> PlutusData
 mkKey str = Bytes $ fromJust $ byteArrayFromString str
 
 lookupKey
-  :: Partial => String -> Map.Map PlutusData PlutusData -> Maybe PlutusData
-lookupKey keyStr = Map.lookup (mkKey keyStr)
+  :: Partial => String -> Array (PlutusData /\ PlutusData) -> Maybe PlutusData
```

Comment:
I'm guessing because it's using `mkKey` which is `Partial`. `mkKey` needs to be unsafe because `ToData` isn't in `Maybe`. On a side note, is there any reason we're keeping this metadata stuff in `master`? @ngua @klntsky

Reply:
Not sure what can be done here so I left it.

Reply:
Ok

Reply:
@vvtran We want to add support for various kinds of metadata, including Seabug and CIP 25. So I think it makes sense to keep the definitions for metadata types in master